azure_functions_worker/loader.py

```diff
@@ -239,3 +242,17 @@ def index_function_app(function_path: str):
                          f"{script_file_name}.")
 
     return app.get_functions()
+
+
+def get_fx_raw_bindings(indexed_function, function_info):
+    # Check if deferred bindings is enabled
+    if (bindings.meta is not None
+            and bindings.meta.DEFERRED_BINDINGS_ENABLED
```

Comment:
todo: remove bindings.meta none check. move sdk_registry check to get_binding & only do once